.github/workflows/winget.yml

```diff
@@ -0,0 +1,55 @@
+name: Release Build artifacts to winget
+
+on:
+  workflow_call:
+    inputs:
+      releaseTag:
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      releaseTag:
+        description: 'Release tag name'
+        required: true
+        type: string
+
+jobs:
+  publish-winget:
+    name: Publish on winget repo
+    runs-on: windows-latest
+    needs: [build-msi]
+    steps:
+      - name: Get download url for msi artifacts
+        id: get-release-assets
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const query =`query($owner:String!, $name:String!, $tag:String!) {
+              repository(owner:$owner, name:$name){
+                release(tagName: $tag) {
+                    releaseAssets(first:20) {
```

Comment:
parameters owner/name/tag were just an example. in this case, I think we can hard code all but the tag.

furthermore `first:20` was also just an example. is it sufficient?

Reply:
Yes, currently it is 100% sufficient, since we have 17 release assets. With more than 20 assets, it depends on the sorting. I tried to avoid pagination and iterating over something.

Reply:
> parameters owner/name/tag were just an example. in this case, I think we can hard code all but the tag.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2471/commits/69747a7a77ffb805485865dbf772e6eed4a7655f